tools/oeedger8r/Emitter.ml

```diff
@@ -1368,8 +1368,13 @@ let gen_enclave_code (ec : enclave_content) (ep : edger8r_params) =
     ; "    const char* path,"
     ; "    oe_enclave_type_t type,"
     ; "    uint32_t flags,"
+    ; "#ifdef OE_CONTEXT_SWITCHLESS_EXPERIMENTAL_FEATURE"
```

Comment:
Yes, @dthaler proposed a snippet that has this definition in `oe_create_enclave`. It would have been cleaner if we have enclave configuration as a separate PR before this PR. But we have to deal with the fact that enclave configuration was not implemented. We have some thoughts, which were never turned into code.